packages/storybook/stories/CallComposite/Quickstart1ToNDocs.stories.mdx

```diff
@@ -0,0 +1,176 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import Quickstart1TonSnippet from '!!raw-loader!./snippets/Quickstart1ToN.snippet.tsx';
+import { SingleLineBetaBanner } from '../BetaBanners/SingleLineBetaBanner';
+import { DetailedBetaBanner } from '../BetaBanners/DetailedBetaBanner';
+import { StartPSTNCall } from './StartPSTNCall.stories';
+import { CallComposite } from '@azure/communication-react';
+import { defaultCallCompositeHiddenControls, controlsToAdd } from '../controlsUtils';
+
+<Meta
+  id="quickstarts-1ton"
+  title="Composites/CallComposite/1:N Call/1:N Get Started"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Getting Started With 1:N
+
+<DetailedBetaBanner />
+
+The UI Library CallWithChat Composite and Calling Composite supports 1:N by enabling developers to perform outbound calls through
+Azure Communication Services call service. Currently, Inbound calls (receiving calls from outbound ACS calls) is not supported by
+the UI Library. However, developers are able to create inbound scenarios using our [Call stateful Client](./?path=/docs/quickstarts-statefulcallclient--page).
+
+The following features are supported:
+
+1. 1:1 outbound calls with ACS users
+2. 1:N outbound calls with ACS users
+3. 1:N outbound calls with both ACS and PSTN users if an alternate caller Id is provided.
+4. [On/Off hold features](?path=/docs/composite-adapters--page)
+5. Removing participants from a call
+
+_Unfortunately, users are not able to join into an existing call through a copy invite link for 1:N and PSTN scenarios at this time._
```

Comment:
I would re word or remove this since invite links wont be supported for PSTN or 1:N since its a different kind of call than a groupCall.